### DISCUSSION TITLE
PI_TXD?

### DISCUSSION THREAD

**Discussion by MikeMcPherson** [alanbjohnston/CubeSatSim#334] [General]
I don't see any references to PI_TXD outside of the schematic, so I presume this is for future capabilities and/or for us to do something with?

Thanks.  Mike

**Comment by alanbjohnston** [comment]
Hi Mike,

On the Beta v1.3.2 STEM Payload board, PI_TXD, which is the transmit serial data line from the Raspberry Pi Zero, is connected through normally closed jumper JP6 to the RXD (receive serial data) pin on the SR105U FM transceiver board.  It allows the Pi Zero to program the SR105U in the rpitx.py code.  This sets the transmit and receive frequency, squelch, PL codes, etc.  This RXD pin is also connected to the Raspberry Pi Pico micro controller, which would allow it to also program the SR105U in a design without a Pi Zero.  Here's how the SR105U is programmed: http://www.kh-gps.de/sr_frs.pdf

The PI_TXD is also connected through normally open jumper JP8 to the Raspberry Pi Pico which could allow two-way communication between the Pi Zero and the Pico in the future (right now it is just one way, from the Pico to the Pi Zero through the PI_RXD pin).

I really need to provide a description of the design beyond the block diagram.

73,
Alan
